### ISSUE TITLE
Strange behavior with the left sidebar in "normal screen mode" and "full screen mode"

### ISSUE THREAD

**Issue by overclokk** [WordPress/gutenberg#29406]
## Description
In "normal screen mode" I don't find the icon to open the left sidebar menu to navigate between pages and templates.
If I activate "full screen mode", open left sidebar menu and deactivate "full screen mode" the left sidebar remain open
without the possibility to close it.

## Screenshots or screen recording (optional)
![2021-02-27 18-44-56](https://user-images.githubusercontent.com/4604932/109395501-0c319f80-792d-11eb-8f81-41c35707ad38.gif)

## WordPress information
- WordPress version: "5.6.2"
- Gutenberg version: "10.0.2"
- Are all plugins except Gutenberg deactivated? "No"
- Are you using a default theme TT1

## Device information
- Device: "Desktop"
- Operating system: "Windows 10"
- Browser: Firefox latest

#29228
@jameskoster

**Comment by jameskoster** [comment]
The simplest solution would be to force full-screen mode (and remove the option) for when a FSE-compatible theme is active.

**Comment by vindl** [comment]
This is a duplicate of https://github.com/WordPress/gutenberg/issues/26125.

> The simplest solution would be to force full-screen mode (and remove the option) for when a FSE-compatible theme is active.

I agree.

**Comment by overclokk** [comment]
@jameskoster Yes but what if someone wants to go to admin page? He|she have to clck at least 3 time, click for opening the sidebar, click on dashboard and click to the admin page (theme, plugin, settings, ...)

I think the simplest way is to also add the admin links to the FSE left sidebar.

**Comment by jameskoster** [comment]
> I think the simplest way is to also add the admin links to the FSE left sidebar.

I think that is a separate discussion that needs quite a bit more thought since we'd have to consider extensibility, but in general I do agree. In fact I'd love to see the FSE left sidebar replace the wp-admin navigation altogether one day.

**Comment by overclokk** [comment]
> I think that is a separate discussion that needs quite a bit more thought since we'd have to consider extensibility

I agree

**Comment by carolinan** [comment]
Hi
Can this be closed, since the full screen is forced since Mar 2, 2021?

**Comment by jameskoster** [comment]
I think so.
